modules/users/client/views/admin/create-user.client.view.html

```diff
@@ -46,12 +48,13 @@ <h3 class="col-md-12 text-center">Sign up using your email</h3>
           <label>Password Requirements</label>
           <progressbar value="requirementsProgress" type="{{requirementsColor}}"><span style="color:white; white-space:nowrap;">{{requirementsProgress}}%</span></progressbar>
         </div>
-        <div class="text-center form-group">
+        <div class="form-group pull-right">
           <button type="submit" class="btn btn-primary">Sign up</button>
+          <input type="button" value="Cancel" class="btn" ui-sref="admin.users"/>
           <!--&nbsp; or&nbsp;
           <a ui-sref="authentication.signin" class="show-signup">Sign in</a-->
```

Comment:
Does this comment need to be here?

Reply:
That's an artifact from the original sign up page. I don't think we'll need it, so I'll remove it.